packages/x-adapter-platform/package.json

```diff
@@ -38,6 +41,8 @@
     "tslib": "~2.3.0"
   },
   "devDependencies": {
+    "@microsoft/api-documenter": "~7.15.3",
+    "@microsoft/api-extractor": "~7.19.4",
```

Comment:
I would use the same version as x-components package, which is the 7.20.0

Reply:
Good point! I've upgraded it.